src/core_plugins/vega/public/vega_view/vega_base_view.js

```diff
@@ -80,6 +67,31 @@ export class VegaBaseView {
     }
   }
 
+  createViewConfig() {
+    const config = {
+      logLevel: vega.Warn,
+      renderer: this._parser.renderer,
+    };
+
+    // Override URL sanitizer to prevent external data loading (if disabled)
+    const loader = vega.loader();
+    const originalSanitize = loader.sanitize.bind(loader);
+    loader.sanitize = (uri, options) => {
+      if (typeof uri === 'function') {
+        // If uri is a function, it must have come from this integration layer,
+        // because user can only supply pure JSON data structure.
+        // A function means that the value can be trusted (e.g. emsfile service)
+        uri = uri();
+      } else if (!this._vegaConfig.enableExternalUrls) {
+        throw new Error('External URLs are not enabled. Add  "vega": {"enableExternalUrls": true}  to kibana.yml');
```

Comment:
Could we use this PR to fix the wording here to `Add "vega.enableExternalUrls: true" to kibana.yml`, to match the actual YML configuration.

Reply:
@timroes `vega.enableExternalUrls: true` seems a bit ambiguous - proper yaml should be a two line value, which would need a different error message formatting.

Reply:
You can perfectly fine place this in one line in yml, since it allows for collapsing nested values into one line. I usually always write my ymls like that :)

Reply:
Bleh, I am so not keen on yaml - too much voodoo :D
Will update, thx.

Reply:
That's what they say about Vega - too much voodoo ;P

Reply:
hehe, fair enough ;)  My YAML gripes are things like `foo: yes` is not a string field, it's a boolean.  In Vega its simply an unfamiliar language that one has to learn. Painful, I know :)